### PR TITLE
[CBRD-24195] Add new test case for user schema feature

### PR DESCRIPTION
**Pull request by zionyun**
http://jira.cubrid.org/browse/CBRD-24195

### CODE REVIEW

File: sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create1.answer [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+unique_name    name    owner.name    target_class    action_definition    
+u1.trig1     trig1     U1     null     insert into [dba.t2] values ([obj].[c1])
```
Comment (youngjinj) [inline sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create1.answer:23]: target_class must be dba.t1.

File: sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create2.answer [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+===================================================
+CURRENT_USER    
+U2     
+
+===================================================
+0
+===================================================
+class_of    unique_name    class_name    owner.name    
+null     u1.t1     t1     U1
```
Comment (youngjinj) [inline sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create2.answer:17]: class_of must be u1.t1.

File: sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create2.answer [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+unique_name    name    owner.name    target_class    action_definition    
+u1.trig1     trig1     U1     null     insert into [u2.t2] values ([obj].[c1])
```
Comment (youngjinj) [inline sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create2.answer:33]: target_class must be u2.t1.

**Review by youngjinj — APPROVED** [review]

File: sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create1.answer (hunk 1 above)
Comment (zionyun) [reply sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create1.answer:23]: In CTP output, object type is displayed as null.

**Review by ssihil — APPROVED** [review]

**Review by sy052 — APPROVED** [review]

**Pull request merged**